### PR TITLE
Csp 1380 get permissions

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPermissionsRequest.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public string? PublicHashedId { get; set; }
+
+    public GetPermissionsRequest(long? ukprn, string? publicHashedId)
+    {
+        Ukprn = ukprn;
+        PublicHashedId = publicHashedId;
+    }
+}
+
+public enum Operation : short
```
Comment (DevenKShah) [inline src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPermissionsRequest.cs:20]: This enum is used in Response, but is hidden in Request object file. 

Also I think we should have a folder within Requests and Response to separate objects out which related to ProviderRelationships

File: src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPermissionsRequest.cs
Comment (DevenKShah) [inline src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPermissionsRequest.cs:1]: This should be under ProviderRelationships folder

File: src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPermissionsRequest.cs (hunk 1 above)
Comment (MarkFCain) [reply src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPermissionsRequest.cs]: moved to own class in repsonses

File: src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPermissionsRequest.cs
Comment (MarkFCain) [reply src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPermissionsRequest.cs:1]: fixed

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**